src/fhirtypes/ElementDefinition.ts

```diff
@@ -2003,8 +2016,8 @@ export class ElementDefinition {
         newElements = this.cloneChildren(slicedElement);
       }
       if (newElements.length === 0) {
-        // If it has a profile, use that, otherwise use the code
-        const type = this.type[0].profile?.[0] ?? this.type[0].code;
+        // If we have exactly profile to use, use that, otherwise use the code
```

Comment:
```suggestion
        // If we have exactly one profile to use, use that, otherwise use the code
```

This might be the smallest comment I could make.